casper/src/main/resources/RevVault.rho

```diff
@@ -33,6 +33,17 @@ in {
       @MakeMint!(*mintCh) |
       for (mint <- mintCh) {
 
+        contract RevVault(@"deployerAuthKey", deployerId, ret) = {
+
+          new RevAddress(`rho:rev:address`), revAddrCh in {
+            RevAddress!("fromDeployerId", *deployerId, *revAddrCh)|
+            for (@deployerRevAddress <- revAddrCh) {
+
+              @AuthKey!("make", (*_revVault, deployerRevAddress), *ret)
+            }
+          }
+        } |
+
         contract RevVault(@"deployerAuthKey", ret) = {
```

Comment:
Well done on migrating all the usages of this method but the one in PoS! :tada: Please add a fat "deprecated, don't use - use the other one instead" comment on this method :)

Reply:
Done